ldclient/twisted_redis.py

```diff
@@ -9,7 +9,14 @@ def create_redis_ldd_processor(api_key, config, store, **kwargs):
     return TwistedRedisLDDStreamProcessor(store, **kwargs)
 
 
-class TwistedRedisLDDStreamProcessor(StreamProcessor):
+class TwistedRedisLDDStreamProcessor(UpdateProcessor):
```

Comment:
Do we need this? It seems like this should be a RedisFeatureStore, not a Redis stream processor..